### DISCUSSION TITLE
how to make number mask use rounding instead removing digit

### DISCUSSION THREAD

**Discussion by laptopmutia** [uNmAnNeR/imaskjs#755] [Q&A]
```javascript
var numberMask = IMask(element, {
  mask: Number,  // enable number mask
  scale: 2,  // digits after point, 0 for integers
  signed: false,  // disallow negative
  thousandsSeparator: '',  // any single char
  padFractionalZeros: true,  // if true, then pads zeros at end to the length of scale
  normalizeZeros: false,  // appends or removes zeros at ends
  radix: '.',  // fractional delimiter
  mapToRadix: ['.'],  // symbols to process as radix
  min: 0
});
```

apparently this code just removing the digit that is more than the scale allow instead rounding it 

so with config in above field that has `value="177.688"` will have masked value as `"177.68"`

any guidance how to make it so the masked value is `"177.69"` instead?

edit: is it because my assumption about removing instead rounding correct? 
or is it because [weird javascript behavior about rounding and math?](https://stackoverflow.com/a/12830454)

**Comment by lbineau** [comment]
I'm in the same case as you. The only difference I have is I'm using directly the number instead of the input element so I can manually round the digits `.toFixed(3)`. I agree it is not ideal and maybe there should be an option of the library. Maybe a `roundDigits: boolean`?
